.github/workflows/performance.yml

```diff
@@ -162,6 +162,12 @@ jobs:
         run: |
           npm run env:cli -- rewrite structure '/%year%/%monthnum%/%postname%/' --path=/var/www/${{ env.LOCAL_DIR }}
 
+      - name: Install additional languages
+        run: |
+          npm run env:cli -- language core install de_DE --path=/var/www/${{ env.LOCAL_DIR }}
+          npm run env:cli -- language plugin install de_DE --all --path=/var/www/${{ env.LOCAL_DIR }}
+          npm run env:cli -- language theme install de_DE --all --path=/var/www/${{ env.LOCAL_DIR }}
```

Comment:
Any particular reason behind loading `de_DE` over other translations? Does it have any particular attributes or just a random decision?

Reply:
Random :)

In the past I tried to see whether a language like Japanese would cause different results but that wasn't the case.